ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -489,6 +489,45 @@ tests:
       AWS_METADATA_SERVICE_AUTH: Required
     workflow: openshift-e2e-aws
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-byo-subnet-role-security-groups
```

Comment:
@yunjiang29 shall we snyc the sg coverage to other versions together ?

Reply:
Updated